File: ML/ambuj-1211_AmbujKulshreshtha_2024csit1160_2/GettingStarted/printz.java


public class printz {

    public static void main(String[] args) {
        int i,j;
        for( i=1;i<=5;i++){
            for( j=1;j<=5;j++){
                if(i==1||i==5)
                System.out.print("*");
                else if(i==6-j){
                    System.out.print("*");
                }
                else 
                 System.out.print(" ");
            }
            System.out.println();
        }

    }
}

File: Web/arjit1704_Arjit_2024csit1132_2/GettingStarted/printZ.cpp
#include<iostream>
using namespace std;
int height = 5;
int width = (2 * height) - 1;
int abs(int d)
{
    return d < 0 ? -1 * d : d;
}
void printZ()
{
    int i, j, counter = height - 1;
    for (i = 0; i < height; i++) {
        for (j = 0; j < height; j++) {
            if (i == 0 || i == height - 1
                || j == counter)
                cout <<"*";
            else
                cout <<" ";
        }
        counter--;
        cout <<"\n";
    }
}
void printPattern(char character){
    switch (character){
    case 'Z':
        printZ();
        break;
    }
}
int main()
{
    char character = 'Z';
    printPattern(character);
    return 0;
}
